app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -37,20 +143,111 @@ export const CollectionsPage = ({ data }) => {
           subtitle="Explore the New York Public Library's diverse collections, including digitized photographs, manuscripts, maps, and more. Start exploring by using the search bar below or browse through the collections."
         />
         <SearchBar
-          sx={{ maxWidth: "462px", marginTop: "l" }}
+          sx={{ maxWidth: "462px" }}
           id={"search-collections"}
           textInputProps={{
             isClearable: true,
+            isClearableCallback: () => {
+              setcurrentCollectionKeywords(DEFAULT_SEARCH_TERM);
+            },
             labelText: "Search by collection title",
-            name: "searchinput",
+            name: "collection_keywords",
             placeholder: "Search by collection title",
+            defaultValue: currentCollectionKeywords,
+            onChange: (e) => handleSearchChange(e),
           }}
-          onSubmit={function (event: React.FormEvent): void {}}
+          onSubmit={handleSearchSubmit}
           labelText={""}
         />
       </Box>
       <HorizontalRule sx={{ marginTop: "xxl", marginBottom: "xxl" }} />
-      <SearchResults showFilter isSearchPage={false} data={data} />
+      <Box
+        sx={
+          collections?.length > 0
+            ? { display: "flex", gap: "xs", marginBottom: "l" }
+            : { display: "none", gap: "xs", marginBottom: "l" }
+        }
```

Comment:
Rewrite this so it ends up with `display: collections?.length > 0 ?  "flex" : "none",` and the other properties are always there (since they're the same in both conditions).